### ISSUE TITLE
JSP competence extraction needs SLM: regex finds provisions but not specific qualifications

### ISSUE THREAD

**Issue by shotleybuilder** [fractalaw/fractalaw#50]
## Gap

385 JSP obligations mention competence requirements (8% of 5,028), but the current extraction is regex keyword detection only — it captures "competent person", "trained", "qualified" without extracting WHAT competence is required.

JSP text often specifies the detail:
- "trained in confined space rescue"
- "CS01 certified"
- "qualified radiation protection supervisor"
- "completed (and remain current in) any training courses deemed necessary"
- "sufficient experience and training in all associated risks"

The regex captures the flag ("competence is required here") but not the specification.

## Current state

Three JSP SLMs were fine-tuned in Phase 6:
- `gemma3-raci` — RACI role classification ✅
- `gemma3-artefact-props` — artefact property extraction ✅
- `gemma3-control-titles` — indicative-mood control titles ✅

Competence extraction was not included. No SLM or training data exists for it.

## Proposed approach

Fine-tune a `gemma3-competence` model:

**Input:** Obligation text flagged as containing competence requirements (385 provisions)

**Output:** Structured JSON:
```json
{
  "qualification": "CS01 certification",
  "training": "confined space rescue training",
  "experience": "sufficient experience in associated risks",
  "revalidation": "remain current",
  "assessor": "competent person"
}
```

**Training data:** Bootstrap from the 385 regex-flagged obligations. Manual labelling of ~100 examples as seed, SLM generates the rest (same approach as artefact properties).

**Value:** Feeds directly into:
- L4 Evidence (competence certificates as artefact type)
- Training needs analysis ("what qualifications do our people need?")
- Role-specific competence matrix (RACI role × required competence)

## Cross-reference value

Combined with RACI:
- Commander/Manager: 10 obligations requiring "competent person"
- Accountable Person: 7 obligations requiring "competent person"  
- Contractor: 3 obligations requiring "competent"

The query "what competence does a contractor need across all JSPs?" requires the SLM extraction to answer specifically, not just "competence is mentioned."

## Priority

Medium — the regex flag is useful but the insight gap is real. Can be included in the next RunPod fine-tuning session alongside any model re-training.
